radar.csv

```diff
@@ -44,9 +44,7 @@
 "Kubernetes",adopt,"Platforms",true,"<a href='https://kubernetes.io/'>K8S</a> is the undeniable winner of orchestration tools."
 "SQL Server as backing store",adopt,"Platforms",true,"When a product needs to store more complex data than it can fit in a flat file, we ask the customer for a SQL DB."
 "PostgreSQL as backing store",trial,"Platforms",true,"If there's a very good reason to use a lightweight/free database engine, we should choose Postgres."
-"Windows",adopt,"Platforms",true,"Windows must be one of our supported target platforms."
-"Linux x64 (.NET supported versions)",adopt,"Platforms",true,"Linux must be one of our supported target platforms."
-"macOS",adopt,"Platforms",true,"MacOS must be one of our supported target platforms."
+"Multi-OS (Windows, Linux x64, macOS)",adopt,"Platforms",true,"All three must be supported platforms."
```

Comment:
I would still include `.NET supported versions` in the `Linux x64` criteria, as described at https://github.com/dotnet/core/blob/main/os-lifecycle-policy.md?WT.mc_id=dotnet-35129-website.

Given the vast number of distros of Linux, sticking to the .NET list of "supported versions" feels like a sensible decision.

Reply:
OK - sure. I've corrected this in https://github.com/red-gate/Tech-Radar/pull/348/commits/f6410c56b578ec29d2ea3422b360c22c45050457